src/hooks/incomplete.ts

```diff
@@ -30,7 +33,17 @@ const hook: Hook.CommandIncomplete = async function ({ config, matches, argv })
     return config.runCommand('help', [toStandardizedId(command, config)]);
   }
 
-  return config.runCommand(toStandardizedId(command, config), argv);
+  const standardCommand = toStandardizedId(command, config);
```

Comment:
nit: use `toConfigureId` to respect the topicSeparator (currently: ' ') as `determineCommand` does.